### ISSUE TITLE
Balance issues on triple shot: No damage penalty? No increased arrow usage?

### ISSUE THREAD

**Issue by lupuleasa-core** [RandyKnapp/ValheimMods#532]
Triple shot is by far the strongest mod in the game, as it simply triples your damage.

Shouldn't the shots do less damage (like 66% per shot) OR use more arrows, or BOTH?

Even at 66% damage per shot (186% total damage) and triple arrow usage it would still be a god tier modifier to get.

Right now it just trivializes the game too much I feel, ranged is balanced by valheim devs by being weaker than melee in general, since you can be safe and away from monsters.

I have insane rolled one handed swords and polearms, and they can barely keep DPS with triple shot bows or crossbows.  While taking tons of damage. This design is not fitting.

Thank you for reading, this mod is my favorite valheim mod, and I love playing it with 5 star creatures and 20% epicloot global droprate for a long and satisfying endgame.

- Loop

**Comment by RandyKnapp** [comment]
I agree. I am wondering if the effect is salvageable or if I should just cut it for now and work out how to balance it later.

Do you feel like the double mage shot one is balanced enough?

**Comment by lupuleasa-core** [comment]
Double mage one is not as bad as the bow one, but I feel that one should get the same treatment (less than 100% damage per shot, make it 75% per shot)

**Comment by lupuleasa-core** [comment]
overall:

- triple shot = 66% damage per arrow, uses 3 arrows
- double magic shot = 75% damage per shot

**Comment by heyufei85** [comment]
Maybe using crossbow can not reduce the damage, but maybe it should increase the loading time?

**Comment by lupuleasa-core** [comment]
Damage seems fine to me, reload time is too slow already

**Comment by Hotair10** [comment]
I would have to agree with lupuleasa on this. My group is looking to disable Triple Shot entirely for our next run, as we are able to run around with bows several tiers below our current biome and just steam-roll starred mobs. The lower accuracy is great, but the enchantment itself seems entirely too overpowering. Lowering the damage per projectile would be a fitting solution to this extremely strong modifier.
